README.md

```diff
@@ -1,2 +1,59 @@
-# node-gsettings
-Node.js binding to Linux Gnome's gsettings API
+node-gsettings
+==============
+Nodejs binding to [gsettings](https://developer.gnome.org/gio/stable/GSettings.html). Gsettings is a common way of storing application configurations on linux in a standardized, transparent manner. Gsettings uses [dconf](https://wiki.gnome.org/action/show/Projects/dconf) for its backend.
```

Comment:
If we want to mention DConf here, it would be better to say:

__In general__, Gsettings uses [dconf](https://wiki.gnome.org/action/show/Projects/dconf) for its backend.